bin/code_climate_reek

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+#
+# Wrapper for the CodeClimate integration.
+
+require_relative '../lib/reek'
+require_relative '../lib/reek/cli/application'
+
+# Map input coming from CodeClimate to Reek.
+class CodeClimate2Reek
```

Comment:
Is there any chance of spelling out the `2` as `To` or am I just old, not hip enough and should lrn2toler8 the barbarity? ;)